devito/core/__init__.py

```diff
@@ -32,8 +33,9 @@
 operator_registry.add(PowerOperator, Power, 'advanced')
 operator_registry.add(CustomOperator, Power, 'custom')
 
-operator_registry.add(CPU64NoopOperator, Device, 'noop')
-operator_registry.add(DeviceOffloadingOperator, Device, 'advanced')
+operator_registry.add(DeviceOpenMPNoopOperator, Device, 'noop')
+operator_registry.add(DeviceOpenMPOperator, Device, 'advanced')
+operator_registry.add(DeviceOpenMPCustomOperator, Device, 'custom')
```

Comment:
Maybe add GPU somewhere in the name?

Reply:
OmpGpu instead of OpenMP? It's same length but maybe bit more explicit?

Reply:
argh, I totally missed this one, sorry. It's green again now, so I'll merge it now and then tweak the name in the next PR. Sorry